### PR TITLE
Argspec for edpm_ovn role

### PR DESCRIPTION
**Pull request by jpodivin**

### CODE REVIEW

File: docs/source/roles/role-edpm_ovn.rst [hunk 1]
```diff
@@ -2,5 +2,10 @@
 Role - edpm_ovn
 ===========================
 
+Many of the parameters exposed by edpm_ovn use defaults derived
+values of other parameters. Take great care while overriding them,
+as changes may propagate further.
+
+
```
Comment (bshephar) [inline docs/source/roles/role-edpm_ovn.rst:9]: We probably have a good opportunity to go through this role and see what might have made sense in tripleo, but doesn't quite make as much sense now. A lot of things were coming from Heat, they might now be easier to change.

**Review by bshephar — APPROVED** [review]
This looks good to me. I was going to mention the true / false params that are labeled as `type: str` instead of `bool`. But maybe that's intentional, I'll let someone from dfg:networking comment if those require attention.

But overall, this looks good to me. I agree with the need to explore potential refactoring here. 

Good stuff.

**Comment by jpodivin** [comment]
recheck

File: docs/source/roles/role-edpm_ovn.rst (hunk 1 above)
Comment (jpodivin) [reply docs/source/roles/role-edpm_ovn.rst:9]: Yes. I've created a task for it.

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by danpawlik** [comment]
recheck

**Comment by danpawlik** [comment]
recheck

**Comment by danpawlik** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Comment by jpodivin** [comment]
recheck

**Pull request merged**